src/test/java/com/dsrm/dsrmbackend/RoomControllerTests.java

```diff
@@ -100,4 +100,42 @@ public void getRoomsInRange() throws Exception {
                 .andExpect(jsonPath("$.content[1].unavailableEnd").value(is(nullValue())));
 
     }
+
```

Comment:
I think it would be good to test if validation works for all fields. You can do it in one test. Just send empty request body.